.github/workflows/sonatype_scan.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. 
+# SPDX-License-Identifier: Apache-2.0
+# Licensed under the Apache License, Version 2.0 https://aws.amazon.com/apache-2-0/
+
+name: Sonatype SCA Scanning
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+env:
+  SonatypeUrl: "https://finos.sonatype.app/platform/"
+  SonatypeAppId: "htc-grid"
+  SonatypeScanTarget: "."
+  ExcludeDirectory: "**/docs/ **/examples/ **/.*'"
```

Comment:
We should include the **examples** directory as it contains actual code that's used in the project. It's named examples as it shows ways customers can implement certain aspects of their code, but it is actually part of the building blocks of the deployment of the blueprint.

Reply:
Done! It's been removed from the exclude list.